### ISSUE TITLE
File numbers returned by partitioning tool

### ISSUE THREAD

**Issue by NoraLoose** [CWorthy-ocean/roms-tools#108]
The partitioning tool implemented in #101 behaves as follows:
```
from roms_tools.utils import partition
file_numbers, partitioned_datasets = partition(ds, nx=1, ny=5)
```
returns the following `file_numbers`
```
[0, 5, 10, 15, 20]
```

Is this intentional @TomNicholas? Don't we want the `file_numbers` to be
```
[0, 1, 2, 3, 4]
```
in this case? My understanding was that the `file_numbers` help with naming the files to write via `xarray.save_mfdataset()`, and ROMS expects  `.0.nc`, `.1.nc`, `.2.nc`, ...

**Comment by TomNicholas** [comment]
That's probably a bug. Looks like I forgot to explicitly check the returned file numbers.

**Comment by NoraLoose** [comment]
Okay, I'm working on a PR that integrates `partition` with the `.save()` methods. I will fix the bug as part of that PR, and ask you to review.

**Comment by NoraLoose** [comment]
Closed via #109.
